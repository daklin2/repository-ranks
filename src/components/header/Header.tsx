import "./Header.style.scss";

const Header = () => {
	return (
		<header className={"header"}>
			<div className={"header__body"}>Github Ranks for UBS</div>
		</header>
	);
};

export default Header;
