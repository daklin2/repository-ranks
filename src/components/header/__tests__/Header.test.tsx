import { shallow } from "enzyme";
import Header from "../Header";
import toJson from "enzyme-to-json";

describe("Header", () => {
	it("should render Header", () => {
		expect(toJson(shallow(<Header />))).toMatchSnapshot();
	});
});
