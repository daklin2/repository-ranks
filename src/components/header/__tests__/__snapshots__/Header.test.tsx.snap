// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render Header 1`] = `
<header
  className="header"
>
  <div
    className="header__body"
  >
    Github Ranks for UBS
  </div>
</header>
`;
